sdk/loadtestservice/load-testing-rest/review/load-testing.api.md

```diff
@@ -9,433 +9,309 @@
 import { Client } from '@azure-rest/core-client';
 import { ClientOptions } from '@azure-rest/core-client';
 import { HttpResponse } from '@azure-rest/core-client';
+import { OperationState } from '@azure/core-lro';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { PathUncheckedResponse } from '@azure-rest/core-client';
 import { RawHttpHeaders } from '@azure/core-rest-pipeline';
 import { RequestParameters } from '@azure-rest/core-client';
+import { SimplePollerLike } from '@azure/core-lro';
 import { StreamableMethod } from '@azure-rest/core-client';
 import { TokenCredential } from '@azure/core-auth';
 
-// @public (undocumented)
+// @public
 export interface AppComponent {
     displayName?: string;
     kind?: string;
     resourceGroup?: string;
-    resourceId: string;
-    resourceName: string;
-    resourceType: string;
+    resourceId?: string;
+    resourceName?: string;
+    resourceType?: string;
```

Comment:
why did these end up as optional? Is it expected that there are no required properties?

Reply:
These are generated by auto-rest from swagger.